features/storage/kvstore/filesystemstore/FileSystemStore.h

```diff
@@ -0,0 +1,229 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef MBED_FILE_SYSTEM_STORE_H
+#define MBED_FILE_SYSTEM_STORE_H
+
+#include "KVStore.h"
+#include "FileSystem.h"
+
+#define FSST_REVISION 1
```

Comment:
These defines don't belong here - they should be in the CPP file (they're all related to your implementation and don't concern the user).

Reply:
Fixed